### PR TITLE
Add support for sku-telemetry

### PR DESCRIPTION
**Pull request by mattsjones**

### CODE REVIEW

File: .changeset/lovely-buttons-shout.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'sku': patch
+---
+
+Add metrics for core scripts
```
Comment (markdalgleish) [inline .changeset/lovely-buttons-shout.md:5]: Might be worth adding a note here to install the package, rather than letting them find out the hard way?

Comment (mattsjones) [reply .changeset/lovely-buttons-shout.md:5]: Was just adding this haha. Is what I added enough or should we add more reasoning?

Comment (markdalgleish) [reply .changeset/lovely-buttons-shout.md:5]: I'd say go a step further and add the commands for both yarn and npm here, so it's literally just a copy and paste.

File: scripts/build-ssr.js [hunk 2]
```diff
@@ -24,10 +27,20 @@ const { port } = require('../context');
     await run(webpack(serverConfig));
     await copyPublicFiles();
 
-    console.log(green('Sku build complete!'));
-  } catch (e) {
-    console.error(red(e));
+    const timeTaken = performance.now();
+    track.timing('build_ssr', timeTaken, { status: 'success' });
```
Comment (jahredhope) [inline scripts/build-ssr.js:31]: Using a seperate key for build_ssr will make it hard to report on 'how many builds of any kind have happened'.
Would it make sense to call the key 'build' and move ssr into a tag?
```suggestion
    track.timing('build', timeTaken, { status: 'success', type: 'ssr' });
```

Comment (mattsjones) [reply scripts/build-ssr.js]: Addressed this for both build and start

**Pull request merged**